function_lab.js

```diff
@@ -0,0 +1,297 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+//
+//
+//
+const returnLargestNumber = (num1, num2) => {
+  if( num1 > num2){
+  return num1
+
+}else{
+  return num2
+}
+}
+
+let largestNumber= returnLargestNumber(12, 4)
+console.log(largestNumber);
+//
+//
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+//
+//
+
+function getDogAge(humanAge){
+  return humanAge * 7
+}
+console.log(getDogAge(4))
+
+//or
+// let getDogAge2= getDogAge(10)
+// console.log(getDogAge2);
+
+//
+//
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+//
+//
+let arr = [13,2,4,-2]
+
+function arraysum(arrsum1){
+ let sum = 0;
+  for(let i = 0; i < arrsum1.length; i++){
+sum += arrsum1[i]
+}
+return sum;
+}
+
+let arraysum2 = arraysum(arr)
+console.log(arraysum2)
+//
+//
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+//
+//
+function sumOfNumber (num1,num2){
+  return num1 + num2
+}
+let twoNumbers = sumOfNumber(14, 5)
+console.log(twoNumbers)
+//
+//
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+//
+//
+function repearLog(num){
+ if (num === 100){
+return "A+"
+} else if (num < 100 && num >= 90){
+return "A"
+} else if (num < 90 && num >= 80){
+return "B"
+} else if (num < 80 && num >= 70){
+return "C"
+} else if (num < 70 && num >= 65){
+return "D"
+} else{
+return "F"
+}
+}
+let grade = repearLog(88)
+console.log(grade)
+//
```

Comment:
Good job, with knowing code control flow... your '&&' statements aren't necessary.
Lets talk about it .